meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -4,6 +4,25 @@ require conf/distro/poky.conf
 DISTRO = "resin-os"
 DISTRO_NAME = "Resin OS"
 DISTRO_VERSION = "1.1.4"
+HOSTOS_VERSION = "${DISTRO_VERSION}"
+python () {
+    ''' Set HOSTOS_VERSION from board VERSION if available '''
+
+    # Find board VERSION location based on poky meta layer
+    bblayers = d.getVar('BBLAYERS', True).split()
+    poky = [layer for layer in bblayers if layer.endswith('poky/meta')]
+    if not poky:
+        return
```

Comment:
Well this is relevant because line14-15 is dead code from my point of view. If anyone removes poky from bblayers (the only case where this code has meaning) then this should maybe throw an error, if the build has not failed already (as poky is a hard dep) in which case this code becomes truly dead.
Not a big deal, just pointing it out.